### PR TITLE
[Feature] AEI

### LINKED ISSUE #657: [Feature] Automated exit interview visibility + option
- [x] Add form creator for exit interview
- [x] Add default form
- [x] Wire up exit interview checkbox in offboarding
- [x] Add exit interview date option
- [ ] Add page for exit interview visibility
  - [ ] Insights
  - [ ] Response view
  - [ ] Response list
  - [x] Form list

### PR DESCRIPTION
**Pull request by James1x0**
closes #657

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -11,158 +14,169 @@ module.exports = {
...
-    'no-return-assign': 2,
-    'no-script-url': 2,
-    'no-self-compare': 2,
-    'no-shadow': 2,
+    'ember/routes-segments-snake-case':           0,
+    'ember/no-attrs-in-components':               0,
+    'semi':                                       [ 'error', 'always' ],
+    'semi-spacing':                               [ 2, {
+      'before': false,
+      'after':  true
```
Comment (JavaLavaMT) [inline .eslintrc.js]: looks like there may be some duplicates in here

File: app/mixins/authenticated.js [hunk 2]
```diff
@@ -22,7 +23,12 @@ export default Mixin.create({
   },
 
   willDestroy () {
-    this.get('auth').removeObserver('authenticated', this, this.__authenticationStateChanged);
+    try {
+      this.removeObserver('auth.authenticated', this, this.__authenticationStateChanged);
+    } catch (e) {
+      // noop
```
Comment (JavaLavaMT) [inline app/mixins/authenticated.js]: eslint fix?

File: app/routes/account/employees/forms/responses.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import Route from '@ember/routing/route';
+import resource from 'granite/mixins/route-abstractions/resource';
+
+export default Route.extend(resource, {
+  modelName: 'form-response',
```
Comment (JavaLavaMT) [inline app/routes/account/employees/forms/responses.js]: title token

File: app/templates/account/job-opening/campaign/job-application/index.hbs [hunk 4]
```diff
@@ -79,7 +79,7 @@
       <div class="item content">
         <h4>{{group.step.label}}</h4>
         <p>
-          {{#if group.response.value.length}}
+          {{#if (is-array group.response.value)}}
```
Comment (JavaLavaMT) [inline app/templates/account/job-opening/campaign/job-application/index.hbs]: nice

File: app/templates/components/modals/form-creator.hbs [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  {{yield openFormModal}}
+{{/if}}
+
+{{#ui-modal name="modal__form-creator" id=modalId}}
+  <div class="header">
+    {{if existingForm "Edit" "Create"}} Form
+  </div>
+  <div class="content ui form">
+    <div class="field">
+      <label for="form-name">Form Name* (for your use)</label>
```
Comment (JavaLavaMT) [inline app/templates/components/modals/form-creator.hbs]: @JavaLavaMT for your issue

File: app/routes/account/employees/forms/responses.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (JavaLavaMT) [inline app/routes/account/employees/forms/responses.js]: title token

File: .eslintrc.js (hunk 1 above)
Comment (James1x0) [reply .eslintrc.js]: @JavaLavaMT ? Eslint doesn't seem to pick them up

File: app/mixins/authenticated.js (hunk 2 above)
Comment (James1x0) [reply app/mixins/authenticated.js]: @JavaLavaMT Basically. Eslint will qualify this an empty block statement, but try/catch needs a catch block even if you just want to ignore the error. In this case, the observer cannot be looked up, so we must "try" to remove it even if it fails. If it fails, we don't care. The `noop` comment fixes the eslint issue but also lets other devs know it's intentional. Like "This page is left blank intentionally" in a test booklet.

File: .eslintrc.js (hunk 1 above)
Comment (JavaLavaMT) [reply .eslintrc.js]: weird, okay

File: app/mixins/authenticated.js (hunk 2 above)
Comment (JavaLavaMT) [reply app/mixins/authenticated.js]: nice!

**Pull request merged**